easybuild/easyconfigs/f/FLANN/FLANN-1.8.4_fix-cmake.patch

```diff
@@ -0,0 +1,27 @@
+# Flann fails to build with newer versions of CMake, see:
```

Comment:
@sassy-crick  please also add the author of the patch? (you :)

Reply:
Done. Added my usual lines so I can be traced back. :-)